src/application/i18n/messages/en.json

```diff
@@ -49,7 +49,8 @@
   },
   "home": {
     "title": "Recent Notes",
-    "authText": "You are not logged in, log in to see your recent notes"
+    "authText": "You are not logged in, log in to see your recent notes",
+    "loadMore":"Load More Notes"
```

Comment:
make it more universal: let it be just `Load More` and outside "home". This will allow using it in other places